config/manager/manager.yaml

```diff
@@ -41,3 +41,6 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "performance-operator"
+  install:
+    spec:
+      clusterPermissions:
```

Comment:
Why does it empty?

Reply:
generated using config/rbac/role.yaml which in turn is generated by the annotations in the controller code.